### ISSUE TITLE
Accurancy in dataset ic13_1015? 

### ISSUE THREAD

**Issue by gzhcv** [Canjie-Luo/MORAN_v2#52]
Hi, what the training strategy(curriculum training or one-stage) was used for training the model you provided (demo.pth) ? 

 The accurancy of demo.pth in ic13_1015(LMDB file download from the link you provided)  is 92.0%. 

Meanwhile, I used the One-Stage training strategy and training with learning rate=1 for 3 epochs，then training with learning rate=0.1 for 1 epoch, all the synth data are used(NIPS2014 and CVPR2016), no other parameter was modified in train_MORAN.sh.  The accurancy in ic13_1015 is also 92.0%. 

How to imporve the performance of the model?(the accurancy can reach 93.2% as you mentioned) Any suggestion?  Thanks a lot for your code.

**Comment by Canjie-Luo** [comment]
您试试再跑两个epoch？我发现不同的机子上测试似乎有点出入。
谢谢您的关注，不客气。

**Comment by Canjie-Luo** [comment]
我稍微调研了一下其他开源的识别代码，也做了测试，例如CRNN和ASTER，似乎MORAN的开源代码和模型是最接近汇报的结果的:joy:

**Comment by gzhcv** [comment]
再跑两个epoch会涨一点点，到92.3%。确实也非常接近您公布的结果了，算下来也就是差了10个样本。不过我用您提供的那个模型(demo.pth)在ic13_1015上测的结果怎么和您公布的不一致啊:joy:

**Comment by Canjie-Luo** [comment]
啊数据集的准确率是代码自动保存起来的。我随便挑了一个模型上传了，怕过拟合到这几个测试集上，遇到其他的样本泛化性能不好，被做实际应用的人嫌弃:joy:

**Comment by gzhcv** [comment]
Soga， 可敬可敬:thumbsup:，学习了
